include/srdfdom/model.h

```diff
@@ -267,6 +296,10 @@ class Model
   std::vector<LinkSpheres> link_sphere_approximations_;
   std::vector<DisabledCollision> disabled_collisions_;
   std::vector<PassiveJoint> passive_joints_;
+  std::map<std::string, std::vector<JointProperty>> joint_properties_;
+
+  // Empty joint property vector
+  std::vector<JointProperty> empty_vector_;
```

Comment:
```suggestion
  static const std::vector<JointProperty> empty_vector_;
```

I'm not particularly fond of this style of reference-return, but we use it in other places as well.
No need for a vector per instance though.

Reply:
Yeah, its not ideal, but I wanted to maintain a way to do this while keeping the method `const` and without copying data

Reply:
The more common C++ian way would obviously be to raise an exception if the entry can't be found.
I trust you considered, but rejected this. Getting an empty vector for non-existing joints is a reasonable interface in itself.

Reply:
Sorry, CI fails because you also need to define the static member in an implementation file.
I.e., in model.cpp add `const std::vector<JointProperty> Model::empty_vector_;`